src/openrct2-ui/windows/MapTooltip.cpp

```diff
@@ -64,6 +64,18 @@ static int32_t _cursorHoldDuration;
 
 static void window_map_tooltip_open();
 
+static Formatter _mapTooltipArgs;
```

Comment:
Effectively, this is still a global. Is there a plan to get rid of this variable eventually?

Reply:
It needs to stay but eventually it would become part of the window.